be/src/olap/tablet_schema.h

```diff
@@ -111,6 +111,8 @@ class TabletSchema {
     inline double bloom_filter_fpp() const { return _bf_fpp; }
     inline bool is_in_memory() const { return _is_in_memory; }
     inline void set_is_in_memory(bool is_in_memory) { _is_in_memory = is_in_memory; }
+    inline bool has_sequence_col() const { return  _has_sequence_col; }
```

Comment:
how about checking sequence_col_idx != -1 instead of _has_sequence_col

Reply:
modified